__tests__/fixtures/lint.hcl

```diff
@@ -0,0 +1,26 @@
+// Copyright 2024 actions-toolkit authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+group "default" {
+  targets = ["lint", "lint-other", "lint-inline"]
+}
+target "lint" {
+  dockerfile = "lint.Dockerfile"
+}
+target "lint-other" {
+  dockerfile = "lint-other.Dockerfile"
+}
+target "lint-inline" {
+  dockerfile-inline = "FRoM alpine\nENTRYPOINT [\"echo\", \"hello\"]"
+}
```

Comment:
based on this hcl definition, we can see no annotation are generated for `lint-inline` target as the dockerfile is inline: https://github.com/docker/actions-toolkit/actions/runs/10166368516/job/28116386475?pr=389#step:10:201

![image](https://github.com/user-attachments/assets/7103e719-4f31-4715-b189-0709bec898b2)